build/Targets/Packages.props

```diff
@@ -47,7 +47,8 @@
     <MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>14.3.25407-alpha</MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-61531-03</MicrosoftMetadataVisualizerVersion>
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
-    <MicrosoftNETTestSdkVersion>15.3.0</MicrosoftNETTestSdkVersion>
+    <!-- Using a private build of Microsoft.Net.Test.SDK to work around issue https://github.com/Microsoft/vstest/issues/373 -->
```

Comment:
Can you point to branch or something that you used to produce the private build? Otherwise this might give the source build effort a problem. :smile:

Reply:
It's from PR: https://github.com/Microsoft/vstest/pull/1301, it hasn't been merged yet.

Reply:
This is meant to be temporary workaround to make testing experience better immediately, rather than wait for a new test SDK build to show up.